config/schema/entity_browser.schema.yml

```diff
@@ -55,10 +55,10 @@ entity_browser.browser.display.iframe:
   label: 'iFrame display configuration'
   mapping:
     width:
-      type: integer
```

Comment:
We should also update values in defaultConfiguration() as they are integers ATM.